website/docs/r/network.html.md

```diff
@@ -50,6 +65,17 @@ This resource exports the following attributes:
 * `location_uuid` - The location this network is placed. The location of a resource is determined by it's project.
 * `l2security` - See Argument Reference above.
 * `labels` - See Argument Reference above.
+* `dhcp_active` - See Argument Reference above.
+* `dhcp_gateway` - See Argument Reference above.
+* `dhcp_dns` - See Argument Reference above.
+* `dhcp_range` - See Argument Reference above.
+* `dhcp_reserved_subnet` - See Argument Reference above.
+* `auto_assigned_servers` - Contains IP addresses of all servers in the network which got a designated IP by the DHCP server.
+  * `server_uuid` - UUID of the server.
+  * `ip` - IP which is assigned to the server.
+* `pinned_servers` - Contains IP addresses of all servers in the network which got a designated IP by the user.
```

Comment:
I would suggest `A list of server UUIDs with the corresponding IPs that are designated by the user`

Reply:
Done